src/screens/walletConnect/walletConnectSlice.js

```diff
@@ -257,10 +260,13 @@ export const walletConnectOperations = {
 					}),
 				);
 			} else {
+				const gasPrice = await EthGasStationService.getInstance().getPrice();
```

Comment:
@maycon-mello maybe we should check if gas price is not in the payload before setting it in the wallet side.